src/wh_13_1/FileOperation.java

```diff
@@ -0,0 +1,57 @@
+package wh_13_1;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.util.List;
+
+public final class FileOperation {
+
+    protected FileOperation() {
+    }
+
+    static void WriteCSV(String fileName, String CSV_HEADER, List<Player> players) {
```

Comment:
literówka : `WriteCSV` => `writeCSV`
CSV_HEADER tez powinien byc nazwan lowerCamelCasem, wtym kontekcie to nie jest stała tylko argument

Reply:
Poprawione